ui/src/assets/Backup.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="backup">
+    <Backup />
+  </div>
+</template>
+
+<script setup lang=ts>
```

Comment:
Should this and elsewhere have quote marks around "ts"?
`<script setup lang=ts>` -> `<script setup lang="ts">`

Reply:
It's not strictly necessary, but I would agree it's better to have it. Fixed on all files.